### DISCUSSION TITLE
Set the range in days of the missed episodes in the Schedule page

### DISCUSSION THREAD

**Discussion by rafi-d** [pymedusa/Medusa#4958] [Ideas & Feature requests]
**Describe the bug**
When you set General->Interface->Missed Episodes Range   - to any # of days - it seems to have no effect.

**To Reproduce**
1. Set it to 8 or 14 & save (BTW - no save indication/confirmation...)
2. Click on 'Schedule'
3. View the different formats/views
4. See no change

**Expected behavior**
Lists type views - should be longer (more days), Calendar view - wider, or multiple raw if weekly.

**Medusa (please complete the following information):**
Medusa Info: | 
Branch: master Commit: f5973b28465246e67465d7d86bdc8aa74d13e659 
Version: v0.2.8 
Database: 44.11
-- | --
Python Version: | 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)]
SSL Version: | OpenSSL 1.0.2j 26 Sep 2016
OS: | Windows-10-10.0.16299
Locale: | en_US.cp1255

**Additional context**
Regarding the Schedule  list-views - I believe this may help:  medusa\server\web\core\schedule.py ->

class Schedule(WebRoot):
    def __init__(self, *args, **kwargs):
        super(Schedule, self).__init__(*args, **kwargs)

    def index(self):
        next_week = datetime.date.today() + datetime.timedelta(days=7) <--- use the above config instead of the "7" days... I have tested it manually and it seems to resolve the issue.

Calendar: seems to need more modifications to revert to multi-raws

Hoping next release will be able to fix this, or at least most of it per the above.

**Comment by medariox** [comment]
This is not what that setting does. It just extends the days from which the backlog starts, it doesn't change the calendar view.

**Comment by rafi-d** [comment]
hmm, so the label that goes with it is not so clear (at least to me): "Set the range in days of the missed episodes in the Schedule page".

It would be nice to have such setting, though... 

Thanks.
